plans/tier1.fmf

```diff
@@ -236,12 +236,6 @@
   discover+:
     test: checks-after-conversion
   prepare+:
-  - name: workaround_kernel
-    how: shell
-    script: "yum install kernel-3.10.0-1160.76.1.el7 -y && yum remove kernel-3.10.0-1160.80.1.el7 -y"
-  - name: reboot machine
-    how: ansible
-    playbook: tests/ansible_collections/roles/reboot/main.yml
```

Comment:
Previously the Satellite repository was not synced, therefore the latest kernel check would fail.
We needed to install the latest kernel available in Satellite repo and removed the official latest.
Now the repository is synced, so the workaround is not needed anymore.

Reply:
gotcha! makes sense to me.